android/src/main/java/com/mattermost/networkclient/helpers/Helper.kt

```diff
@@ -73,18 +79,12 @@ fun Request.Builder.parseOptions(options: ReadableMap?, session: OkHttpClient.Bu
 fun OkHttpClient.Builder.parseOptions(options: ReadableMap?, request: Request.Builder?): OkHttpClient.Builder {
     if(options == null) return this;
 
-    // Following Redirects
-    if (options.hasKey("followRedirects")) {
```

Comment:
Why is this being removed?

Reply:
It exists in `SessionConfiguration` - this is just cleaning up from an earlier change.